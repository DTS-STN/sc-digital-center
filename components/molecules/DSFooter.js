import { Footer } from '@dts-stn/decd-design-system'

export default function DSFooter() {
  return (
    <Footer
      lang="en"
      btnLink="/"
      logoUrl="https://www.canada.ca/etc/designs/canada/wet-boew/assets/wmms-blk.svg"
      logoAltText="stuff"
      error={false}
    />
  )
}
