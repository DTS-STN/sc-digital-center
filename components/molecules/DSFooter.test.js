import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { axe, toHaveNoViolations } from 'jest-axe'

import DSFooter from './DSFooter'

expect.extend(toHaveNoViolations)

describe('Design System footer tests', () => {
  it('renders DSFooter', () => {
    const { container } = render(<DSFooter />)
    expect(container).toMatchSnapshot()
  })

  it('has no a11y violations', async () => {
    const { container } = render(<DSFooter />)
    const results = await axe(container)
    expect(results).toHaveNoViolations()
  })
})
