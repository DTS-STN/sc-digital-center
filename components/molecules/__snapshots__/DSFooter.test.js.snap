// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Design System footer tests renders DSFooter 1`] = `
<div>
  <footer
    class="ds-w-full"
  >
    <div
      class="ds-bg-multi-blue-blue70 ds-landscape ds-bg-no-repeat bg-clip-border sm:ds-bg-right-bottom ds-bg-bottom"
      style="background-image: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNjE2IiBoZWlnaHQ9IjE4OSIgdmlld0JveD0iMCAwIDYxNiAxODkiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiPgo8cmVjdCB3aWR0aD0iNjE2IiBoZWlnaHQ9IjE4OSIgZmlsbD0idXJsKCNwYXR0ZXJuMCkiLz4KPGRlZnM+CjxwYXR0ZXJuIGlkPSJwYXR0ZXJuMCIgcGF0dGVybkNvbnRlbnRVbml0cz0ib2JqZWN0Qm91bmRpbmdCb3giIHdpZHRoPSIxIiBoZWlnaHQ9IjEiPgo8dXNlIHhsaW5rOmhyZWY9IiNpbWFnZTBfNjIwXzM2NiIgdHJhbnNmb3JtPSJzY2FsZSgwLjAwMTYyMzM4IDAuMDA1MjkxMDEpIi8+CjwvcGF0dGVybj4KPGltYWdlIGlkPSJpbWFnZTBfNjIwXzM2NiIgd2lkdGg9IjYxNiIgaGVpZ2h0PSIxODkiIHhsaW5rOmhyZWY9ImRhdGE6aW1hZ2UvcG5nO2Jhc2U2NCxpVkJPUncwS0dnb0FBQUFOU1VoRVVnQUFBbWdBQUFDOUNBTUFBQUF3WFhIT0FBQUFlMUJNVkVVQUFBQVlKamdiS0RvZ0lEa2NPVGtnTURnb0tFTWRKemdkS0RjQUFBQWdLVGtmTGo0Z0tEZ2ZLamdlSmpnZUtqZ2NLRGdmSnpoQVFFQWZLemNlS0RnZEtEY2RLamNBVlZVaE4wTWVLRGNlS3pjQUFGVWRLamdjS3prZUxUd2FNek1kS2pjZktUWVpKakljS3prYUtEZ2NLRGdZS2pnYktUZ1lLRGpvOS9nd0FBQUFLWFJTVGxNQVNUa29DU0FUZS84QlVDSE5TV1hjdDFzRVUrRDluUU1YZUlvRDh6WWlDc04xUFNUL2JaSzIyMU9CamhRQUFBOHFTVVJCVkhnQjdOVlZBVU5SREFXd1ByNWpacDUvandNUi9VczBIQWdBQUFBQUFBQUFBQUFBQUFBQUFBQWdTMVUzVFJ0L1hkTkhFcWlHVXVyUmVCSXhMVU9RU05CbTg4Vnl0UmEwWklLMjJaYmQvaUJvMlZ6bjhYUytoS0NScURwZHI3ZTRQeUtlMTFja2dlcjlaZGMrYkdVTFlTQU1qMnVaTWFiL0JsL21Kb1dYTGl1dk5WOEhpSC9QWWdTR01JZG1GdmhqWmc3TkhKb1pFY1J0Wm1RRWJqT2o1TkRNb1psRE0zTm8xa3hJaWR2TVVsb3d1eTJsZ3RsdFM5cUUyV1VsVHdOMkg3ZkdUUU1rckJ0cVlHaEJXRE1oalJzN0tjS2FTV25jMkJrT3JaOGxTUnVqcEFMV01qUU9XMVBDbWlsOUY4UFd0TkNRYnpla0hMYW1ndlVTbWhkYTlEc0xXT3FIbXJhbWdIVU1iVStiYnhMV3lwTEdqWjNWN3dyYVNqL0ZzUG1tMEpDSFRpbUh6VGM3WUkyRTVvV1dqUmZrME5hMDBBS05XT3FYY21nWDJaTHVqNTBPelVvSGgvMTRFbTFZNU5hUmdTR3EyNkhUVW0vVXJCdWJUWmhEZThDaUFoZVpRMXVQdUJrMGh4YVBHS1BOb2JIWmNpejRQalNtdjJnWFdPbURUUThEZHVNYTRLT1lGRnFpQlFzZFZYdlEzbFRQWndKK3hMMUl4SnpiZE81M3c0MlJQVUpiUUFDc0tYc1R6VTZjRG0zcGJFZWMvZGw0ZnZuK3hHbU1aSVBRQ2o5RDQ1VFFsbzZFZ1ZuS0RqdXlmalEvS0xUU3NXQUFJN1BMWHllQ0kvNDY0ODBzY0U2Y1JBcytvK2xuN2x3RFFzdHoxbnd6RFNSNmNHaDdFWWlsSVYrMDFLdG9GSnBEazNiVmxvWjgwWlplSmNCRUV3N3RlUDdRaWtEcDFRSmkrYmwrRXAvTm9YSHJWUUZadnQ4SWgvYTV2ckozSjhpSkswRVFRQ3ZQa3BsVmRmOFQvckJGQjYxdllZU3RNV0NUTVlBMmI4MmoxUXRpMkE1clN1TUZMY0tNT3daVnZRR3RLaG5QR21wQUcwSGtheUlLOTRWbTVBWTB3NHhuRFZTUm1wTlJyNG1vUk53My9tMm5UcXRabWxQc0Y3UVh0S05qYmNXdlhpZGUwQTVOdnFCdGhESWZBRnE1eXRudjk4OE9yYlNWK3ZQUUtoNEFHb0xqSCt1NW9iRzFsV2JjUDY4YWpYRk9xbjlkRSszMS9rZi9ST3ZoQmUzVlNLc1h0Qjlvb3IwYWFVakplRUU3TG1odHB4Ri9PTjVUcVR2OGduWjdoZFpsemNuNHUyRzlvUDJUUG1kWEdrRXRzVjEvdTkrSlhaL21XZDB2YUh1VEduOFBzN1dFb3kzOFoyTVQxNnUwVXIyZzNkYnBCSnlsa2U3dVB6Mit3VUtBMTRiUzJHcStvTzBNdEowL0RRMUNJQ2o1V3RFNXZwb1h0SGIvZFdpNVFDTXlMb2VXbEl3YjhvSldUSTA0OG85RFk1RkVJR0IrN2t3eS95bTBtdEsvQUJyQ3RFN21ndjIzb1VFSU1CQ1FyNTRLL0hxYjBBM1FHQXlmQ3k3L01EUkdwRHlnWlFUdkNxME0xL3ZkbmFCaHZmSnRhREdnWVN6K1ZXaGt5ZUVJQkdIeXZ0QVFFWEJFc080RHpldVZZNkNOUmZ4bGFKS0lCVnFBS2R3VkdnZTB1Qk8wNUx6eWduWllxS0pqUUFzTHNSVkNPcm8zQUQ0Z3RNSjVtZlVOc0JlaE9mNWtvSXdGbWhubUpXalFpT09vMkk4SGplMFlDVGUvRHZZaXRQcXIwSXBGTnFJY0tmMXhhRkRPdnhJT2FBbmtxMGFiQlJtZFVSZkxBVkxhNmQ4T3pYT0ZrL0tyamZZUGhxOGEvUmswUkFSL083UlVjeXl6bFM5b3gwSnJkN3NxKzk3UUFNQUpBUGVCVmhNRVN2MkNkaUMwWm1aVWhTdnFYMExqdUYvQ0xueUE1cmVrZmFlWkFjNnZORWpOeDRCR2dQRjBzV09FQVNuRGlISWd3eEwrR1RTUCt5VXAxYVZUSnhab3hUdTBXSFAralhCRVorQUFhQkxqNlFMRUVocGh5WkdNZExBQ2xqZm4xaUZWWnRhM29QRXR1NkNsbHVBT0RZbWFoOEw4R05BczVSUFhhQlRZU2taRmxDT0trU3BDdk5EcnRPc2IwRnB2YWU2QVZscmlPMEJMVHRCeWE4NHVkZ1JIUW1OTHhYaTJaSzRLbzRLNVFMTkhrM3o3MUltSS9EWTA3WURHUHZxS0llKy9TZ2R6OHhVYkZuRmNHNDJ4STZ4ZUdoTHhYR0hWcWpBYzlnbGFqdXVoN2d2dEgxeHRtL3Q3blp3YVJyVkZzV0l6NE0zUWlKdUdPaGxQRmFpNUxveEV2Ti9ZRE8rcDBiaG9JL0dGU3gyNXZKQlJVdUVLdEdaOFArUTdOQjhBaldHcEhkeXNORytHQnQ4MDFJbDRxaXc5eTNQcmdXeVNDWklDS1VuNUNiUTBneDBSa2IyY1dqamN4aEw2dE03ekl5TFFvN2pZQ0w2djFTVm9SNVl0SENoR0hRRE5XWklxdDRBVWJtNmp1WDR6dEpTV2tvK2wwTElxejNmVmt2b1RhTktBVnBJU2dJMzNKTjVDdVBpZXpPV3hFSkVScVNVWmtJUGFCYzN4L2RqVmF0ZnVOdHJsY1RSZmJqdXlmVE8wYXV6RHY4UlBkMVY2QmV2Y3JiTGZsbjJxeXBGUzdZSG0xczUwTW9rZWF6eEQ2L2JuMFBLNFdiWTZZSGpEbDl1T1Z0MDZxYzdlWTRlZGxxUk1NNTRvWEo3cmlrQ0R6WWppZ0xid0l3dWZRYXRnQjdQMTlWVG1rUFE1dElydnAzWi91Z3ByT280dHFiZ0pyUm5nNVZuU3dVc2dVSHBQdm1VTnpiditQbVJ3K1RyaUNXZlJteXlIdmVoYW9DM21rT0hpSjlBYTdDb2RrMjFvUi9ZR3NIK3NCSm9rYkkzMHBFNUJKRGNucjBhODR3TGkycU1mVmNTcHlaSEpaL3dNM3lLTEVmWVoyZ0tQa2I0SWJlUm5vTWsyRC9xRGkvdWc1YXBPaWlrc25XSzJQdzcxZm5ZQmNiMUZrakRibEhlL1RpcHN4Qk5DYXkrZUVtZG95SWh3Qmh2M2h6YUNvejVMMXZ0cSs1b0x5bkZKano4VWEvTXl0STNPZ1BlMURXWm9UL2tHdEFvMlk3a2IwRmp2ZHd3WGZ3WmFmWHozUmg0S3pScFIzUVN0UHB4djJSZWdqVjNZRDIxSWJ2eDZhRmh3T2VNTUxaTExKcFovQ0ZxZVVyb1VIUFVSbjQzYm9Ybmp6TG1rL09FRTUwK2hjUVZ0c00xZkQ2MnJHUkdKR1pvZFN5V0hSa3pKNDZGdDVYaG9icDJUKzl0b201MkIxQngvOE14UG9lV0F0anFpK2N1aGpaSXF6dENRWTlvOWF5NjJmbFpvcVNuTlBkRDhCV2krWHFPNXNZS1dWLzg4Y2dXTnp3VU5HaWxPYitVb1R4TXBhSVBLR0dIcFNhR3h0MndjWDZOUmI4bTluNCtHU3RaMk5ldnlXTXgyZUs2Uzh6bWhlU0YxQWpiQTFmdTlpdTd5VkFzT2FPWE0wVjdQc2R5bmhYVFhlcjlHMUpuTHYvcEJhTllxeFgzUTNOMDkrdVo3b0dIVW1QdkcwV1k5N1ZWU2cxT1dIUDIwbitqdGRZVTJpcXRybm43RDIycU5QNWs5UWNzQWhsYU01UXB3NlREVWVqOVh6d3NqT0NaaTBFdlJXV04xUEM1dU0xM08ram8wQWhHMUgrME1EWE10NHZSV1owQmQ1N1FrTUx1NnE5N3UrbE5vMFR0R2ZFb3lXM1BxMldxMFpMVlhiK1BySEE2NTFOb01kR0ZaMFF3TjlrbUhQSmJydEdCVXJmZHZWQUJZbE5jWVkxbXFndFR5V0V0NUVoR0l3SmVob1ZTMzFXaFRyeE16cDFaWGRYVzNQZzBpdFNNZ3lkYmxsQWUwRDJrK1dZM0djTWRpaU9PU3BJaEpIb1NJSEtmWEdkck4yWVoyK2xZWlVXTjFQQzdRL0Mxb3JKSk1VcXM0OWtDRFN6ZG5RRHNnbmJ3QVRjOEhMVE5Jb2dUQWRxcjlGbGpKdDhnUkhMUEN6d2N0djlUY1MzMG5SdW1USk9xMjBjWGUyc0huZ29id3RUTkJWNVZHam9DV1BpVWxqVDVCWmZaWVBmL1FBcHlBQVI4S3JYazB0UDBwQmt2Znp2TkJVNlh0VWdJQXJTTGVZdGwydG1vTjhibUdOM2dCV3R3UG1pUEMra2E2K3FtZ1lmbXJmUnF1RmRhZm9rTmh0TjJJMUJURFFPbEgwdFhWTjBJRHBtWGRBVnF4cjFSRjBNV1lMVFd0T1kxcFBRUDlkTkJ3R1ZwalFJdndzWGhxdkN5dkpnTlJSZEI3b2RGbGcyUDVTR2lWV1ZsVldlZGZ2cXBXdDdkazlNYmV6NkUxT1hhWGxBSE5xZURzT09vWlQ1MERHdGZRWWd4N1hJWldyckhZNmV6VFF2V3kwL09HK1pBZ3U0S0xkQkNXaWdCTFRZQTVWc2RIV0ZmYzBCa1l3M0VrMHRJeDBFeGE4c2NaZWpQSTZaYVNUcU9yeWRVZVhvRVd1NkZwQnpROEhyUzhCZzJYcXZyaUFLajJmTjJjVzBwR2NON1FZMEY5R2hiR1ZPeTVQTDAxWC9hZjE2REJoWmpEalViWk1kQWkxdER5dkdPRVkvdUFGcXY4S0xUc1I0Tkd0QndSN1A0L3RNcmxxT0lsYU9mdE9iNmpWOWVxY1d6STg4SVpHcjhIalU2SjZ3Mmd0U2ZOR0NIU1Y2RlZWVWs5WmV4WVoxVG12YkhyTXJRVnBYUko1ZFNjY3Q0Q3phMEhuSUphb0tsakE5ckF3M0VDQVE2RE5tYWdBRXNGQUtVR2dCeXJ5NDlrRmttdm9ZMnFDNGtBVFNDUTJoL08zd1kvMnV2Y2JvcFVXbnRUMVZlZ1NmRXdRYzhUdEQ1RHF3R3Rsc1BXLzZjTTkwREx6NkdOZkd0NGcxcVNKN0JaWC9sV2Jra3dnalNKYTUyQlZtZTkzNlJ5alcybjI4aFlIaWpLZFJYYWtlTm96QWVDNXA2YlZ5ejUvMWNsdW4yU2xoSHVBZTM3TlpveU82dXpEb0ZtV3JjbjE3UGpSVXFTMTlETTFoS0FIajBueXdFdGEycEdqZTBhV2ZhcnlLNWxmMzRKV2xkZmgwWXNSVzF5SGs5NE9HZktwYnliQTlwWU9rKzNONWFpOTFzT3FORVlEanFna1hPdkV6QkxTcDc4Tk5CbUpsTVNHT2RZMzBsdVZvdUZOYlNvTTNGSURBakI5OXV5cGdxNkdSYURHc0d5bkJHWkVSWTlneUZXdjN5eUIrcW1OZEswS25MeW5xd0JlUnBIUXlwUDdWVmo2Wk80N3dtTnhMU1N2V29XVzhxUGw3K1dpaEhCdmpCclVuYmVDczFMRzgwUmEyZzUvYy9HT2RXZEZWRUlZemw0ZnRYV2hwNnNMMEtyVTRIY0JvMVNaYWtqTEh3Q2JiTUxPNExJNWFGV2Zjc01xc2Y3d21zY09KNnlBUzJ5SkpEUVc4b0V5NlU3UXFOVjJKcVg4ZWtFNG84WDlLZUVzeEpyTTkrQXh2OS9nN3dHVFVvemdzU1dNeUdzblNraklnaXRVN3dJalZ2UTNuSXp0TUs4alpIakljUGIwSEwxMTNWd1FNdldoY1I5Z3BSRWJqUjF4MVBxYzQyR2lKVU9TODMzZTl0dXpmbkpHbTFReTlSMlRHcDNFTnh3U2VLV1UrZW9ZU3dFOWtMVFJXaVRtK0puMExRVU1Fc1BCVzM4UXBVa1FkUm9CRWc5Um5VY0xPUDBMQzhaOVJ3azVUdTBaZXVxRjdZazhaYVVqTGZVYWNIbkRlZEQxRUFDeGczUURPdmdtT25TeHlRSE5KMFBTTnVTblVwYjZkU3lOdmJXcW1lWjlxbHBVVFdPbXRQY2hMWngzRFZvZWpSb1dqTFh6ZlcrWEVNU0FsS2FFN1Njb0JWVDZtRjJlaVVmbEs2cTZ2R2dlc3RwZTFmMU1XOEUyNWNCN1lleURXMWtFeG9ITkQ4dU5HakVZYWxCQmkzSjVvY3hKc21ycmtPVHFNMWU1NFBFQVgwN25vYzNOaDJYUzRjRk4wSWJJZUY2TkdqKzJDMHpGMGo4V0dyNDhLbGRwVG5maFZha3BTUlRCeWVEK25aeW80Mm1KdFFNSCtkNXhLN2JvWTA4QURRbXlMa3JzRDFOelBwa1hQUndhT3N1WEVBSHg1bi9DbHBBTlFvU3BJNE9YTmVnOFJHaHdXUFlpVUIvL3RhTGk5QnFRT3NuZ1haSTB0Nkcxb1R1a2FucWZ6aG80MWV6bmRyT2RXZ3NKZjhlTkttOGVlcDhxc1FQaExGdGFPL29KaGtSWENyRmRFVHdBalRBdnd6YS9rN3NDeG96c2FlQU1sSVgwbW1qUnM4cmdqbzAveDZhWG9sL0hMb091WUxpSEFkOU5EUlNBdUgvMnJ0WEk0dGlJQWFpZGl6U2ZQS1BjR3ZCcFE5WnJBK1pCTHFHeWtnNVllWFhwc2R2N2E0Umx2OW5TU0MwSzUyN2pnS2hhZHcxVGdDaDZidVNZMEJvZFd0dVNlMGNFTnI5eGx2Yk9TQTArVi9YSGNlQTBEUk9BNkhwdE5OQWFKb3FoNEhRMUU0Q29hbXFaOWM1SURUVnJNTkFhQnJIZ2REdU9nK0UxczREb1dtY0JrSlRyZE5BYUdybmdkRGFUNERROWpjL0FVTHpNOEFmMnFMOTh1NzF0MUVBQUFBQVNVVk9SSzVDWUlJPSIvPgo8L2RlZnM+Cjwvc3ZnPgo=);"
    >
      <section
        class="ds-container"
      >
        <nav
          aria-labelledby="accessibleSectionHeader1"
          class="ds-pt-6 ds-pb-6"
          role="navigation"
        >
          <h2
            class="ds-sr-only"
            id="accessibleSectionHeader1"
          >
            About Government
          </h2>
          <ul
            class="sm:ds-grid sm:ds-grid-cols-3 ds-flex ds-flex-col ds-gap-1 ds-text-xs"
          >
             
            <li
              class="ds-text-white ds-w-64 sm:ds-w-56 lg:ds-w-80 ds-my-2.5 hover:ds-underline"
            >
              <a
                class="ds-text-multi-neutrals-white ds-font-body ds-text-browserh7 ds-leading-20px ds-font-regular hover:ds-text-multi-neutrals-white hover:ds-underline"
                data-cy="[object Object]0"
                href="/"
                id="[object Object]0"
              >
                <span
                  class=""
                >
                  Contact us
                </span>
                <abbr
                  class="ds-language-toggle-abbr"
                  title="Contact us"
                />
              </a>
            </li>
            <li
              class="ds-text-white ds-w-64 sm:ds-w-56 lg:ds-w-80 ds-my-2.5 hover:ds-underline"
            >
              <a
                class="ds-text-multi-neutrals-white ds-font-body ds-text-browserh7 ds-leading-20px ds-font-regular hover:ds-text-multi-neutrals-white hover:ds-underline"
                data-cy="[object Object]1"
                href="/"
                id="[object Object]1"
              >
                <span
                  class=""
                >
                  News
                </span>
                <abbr
                  class="ds-language-toggle-abbr"
                  title="News"
                />
              </a>
            </li>
            <li
              class="ds-text-white ds-w-64 sm:ds-w-56 lg:ds-w-80 ds-my-2.5 hover:ds-underline"
            >
              <a
                class="ds-text-multi-neutrals-white ds-font-body ds-text-browserh7 ds-leading-20px ds-font-regular hover:ds-text-multi-neutrals-white hover:ds-underline"
                data-cy="[object Object]2"
                href="/"
                id="[object Object]2"
              >
                <span
                  class=""
                >
                  Prime Minister
                </span>
                <abbr
                  class="ds-language-toggle-abbr"
                  title="Prime Minister"
                />
              </a>
            </li>
            <li
              class="ds-text-white ds-w-64 sm:ds-w-56 lg:ds-w-80 ds-my-2.5 hover:ds-underline"
            >
              <a
                class="ds-text-multi-neutrals-white ds-font-body ds-text-browserh7 ds-leading-20px ds-font-regular hover:ds-text-multi-neutrals-white hover:ds-underline"
                data-cy="[object Object]3"
                href="/"
                id="[object Object]3"
              >
                <span
                  class=""
                >
                  Departments and agencies
                </span>
                <abbr
                  class="ds-language-toggle-abbr"
                  title="Departments and agencies"
                />
              </a>
            </li>
            <li
              class="ds-text-white ds-w-64 sm:ds-w-56 lg:ds-w-80 ds-my-2.5 hover:ds-underline"
            >
              <a
                class="ds-text-multi-neutrals-white ds-font-body ds-text-browserh7 ds-leading-20px ds-font-regular hover:ds-text-multi-neutrals-white hover:ds-underline"
                data-cy="[object Object]4"
                href="/"
                id="[object Object]4"
              >
                <span
                  class=""
                >
                  Treaties, laws and regulations
                </span>
                <abbr
                  class="ds-language-toggle-abbr"
                  title="Treaties, laws and regulations"
                />
              </a>
            </li>
            <li
              class="ds-text-white ds-w-64 sm:ds-w-56 lg:ds-w-80 ds-my-2.5 hover:ds-underline"
            >
              <a
                class="ds-text-multi-neutrals-white ds-font-body ds-text-browserh7 ds-leading-20px ds-font-regular hover:ds-text-multi-neutrals-white hover:ds-underline"
                data-cy="[object Object]5"
                href="/"
                id="[object Object]5"
              >
                <span
                  class=""
                >
                  About government
                </span>
                <abbr
                  class="ds-language-toggle-abbr"
                  title="About government"
                />
              </a>
            </li>
            <li
              class="ds-text-white ds-w-64 sm:ds-w-56 lg:ds-w-80 ds-my-2.5 hover:ds-underline"
            >
              <a
                class="ds-text-multi-neutrals-white ds-font-body ds-text-browserh7 ds-leading-20px ds-font-regular hover:ds-text-multi-neutrals-white hover:ds-underline"
                data-cy="[object Object]6"
                href="/"
                id="[object Object]6"
              >
                <span
                  class=""
                >
                  Public service and military
                </span>
                <abbr
                  class="ds-language-toggle-abbr"
                  title="Public service and military"
                />
              </a>
            </li>
            <li
              class="ds-text-white ds-w-64 sm:ds-w-56 lg:ds-w-80 ds-my-2.5 hover:ds-underline"
            >
              <a
                class="ds-text-multi-neutrals-white ds-font-body ds-text-browserh7 ds-leading-20px ds-font-regular hover:ds-text-multi-neutrals-white hover:ds-underline"
                data-cy="[object Object]7"
                href="/"
                id="[object Object]7"
              >
                <span
                  class=""
                >
                  Government-wide reporting
                </span>
                <abbr
                  class="ds-language-toggle-abbr"
                  title="Government-wide reporting"
                />
              </a>
            </li>
            <li
              class="ds-text-white ds-w-64 sm:ds-w-56 lg:ds-w-80 ds-my-2.5 hover:ds-underline"
            >
              <a
                class="ds-text-multi-neutrals-white ds-font-body ds-text-browserh7 ds-leading-20px ds-font-regular hover:ds-text-multi-neutrals-white hover:ds-underline"
                data-cy="[object Object]8"
                href="/"
                id="[object Object]8"
              >
                <span
                  class=""
                >
                  Open government
                </span>
                <abbr
                  class="ds-language-toggle-abbr"
                  title="Open government"
                />
              </a>
            </li>
          </ul>
        </nav>
      </section>
    </div>
    <div
      class="ds-h-full ds-pb-4 ds-bg-color-multi-neutrals-white "
    >
      <section
        class="ds-container ds-h-auto ds-pt-5 ds-flex ds-flex-col md:ds-flex-row ds-justify-between"
      >
        <nav
          aria-labelledby="accessibleSectionHeader2"
          class="ds-mt-3.5 xl:ds-mt-5"
          role="navigation"
        >
          <h2
            class="ds-sr-only"
            id="accessibleSectionHeader2"
          >
            About this site
          </h2>
          <ul
            class="ds-flex ds-flex-col sm:ds-flex sm:ds-flex-row"
          >
            <li
              class="ds-pr-4 sm:ds-mb-4 ds-mb-5 ds-list-inside ds-list-disc sm:ds-list-none ds-text-xxs"
            >
              <a
                class="ds-underline ds-text-multi-blue-blue70b ds-font-body ds-leading-20px ds-text-browserh7 hover:ds-underline"
                data-cy="0"
                href="/"
                id="0"
              >
                <span
                  class=""
                >
                  Social media
                </span>
                <abbr
                  class="ds-language-toggle-abbr"
                  title="Social media"
                />
              </a>
            </li>
            <li
              class="ds-pr-4 sm:ds-mb-4 ds-mb-5 ds-list-inside ds-list-disc ds-text-xxs"
            >
              <a
                class="ds-underline ds-text-multi-blue-blue70b ds-font-body ds-leading-20px ds-text-browserh7 hover:ds-underline"
                data-cy="1"
                href="/"
                id="1"
              >
                <span
                  class=""
                >
                  Mobile applications
                </span>
                <abbr
                  class="ds-language-toggle-abbr"
                  title="Mobile applications"
                />
              </a>
            </li>
            <li
              class="ds-pr-4 sm:ds-mb-4 ds-mb-5 ds-list-inside ds-list-disc ds-text-xxs"
            >
              <a
                class="ds-underline ds-text-multi-blue-blue70b ds-font-body ds-leading-20px ds-text-browserh7 hover:ds-underline"
                data-cy="2"
                href="/"
                id="2"
              >
                <span
                  class=""
                >
                  About Canada.ca
                </span>
                <abbr
                  class="ds-language-toggle-abbr"
                  title="About Canada.ca"
                />
              </a>
            </li>
            <li
              class="ds-pr-4 sm:ds-mb-4 ds-mb-5 ds-list-inside ds-list-disc ds-text-xxs"
            >
              <a
                class="ds-underline ds-text-multi-blue-blue70b ds-font-body ds-leading-20px ds-text-browserh7 hover:ds-underline"
                data-cy="3"
                href="/"
                id="3"
              >
                <span
                  class=""
                >
                  Terms and conditions
                </span>
                <abbr
                  class="ds-language-toggle-abbr"
                  title="Terms and conditions"
                />
              </a>
            </li>
            <li
              class="ds-pr-4 sm:ds-mb-4 ds-mb-5 ds-list-inside ds-list-disc ds-text-xxs"
            >
              <a
                class="ds-underline ds-text-multi-blue-blue70b ds-font-body ds-leading-20px ds-text-browserh7 hover:ds-underline"
                data-cy="4"
                href="/"
                id="4"
              >
                <span
                  class=""
                >
                  Site Privacy
                </span>
                <abbr
                  class="ds-language-toggle-abbr"
                  title="Site Privacy"
                />
              </a>
            </li>
          </ul>
        </nav>
        <div
          class=" "
        >
          <div
            class="ds-mt-3 ds-float-left sm:ds-hidden"
          >
            <a
              class="ds-flex ds-flex-row ds-btn-link focus:ds-ring focus:ds-ring-offset-4 undefined "
              href="/"
              id="top_btn"
              role="button"
            >
              To the top
            </a>
          </div>
          <img
            alt="stuff"
            class="sm:ds-h-40px ds-h-25px ds-mb-2.5 sm:ds-mt-6 md:ds-mt-3 ds-mt-5 ds-float-right ds-undefined"
            src="https://www.canada.ca/etc/designs/canada/wet-boew/assets/wmms-blk.svg"
          />
        </div>
      </section>
    </div>
  </footer>
</div>
`;
